### PR TITLE
core(load-fast-4-pwa): use computed artifacts

### PR DESCRIPTION
**Pull request by patrickhulce**
means we'll never pass with the caveat :D we can always answer the question!

### CODE REVIEW

File: lighthouse-core/audits/load-fast-enough-for-pwa.js [hunk 1]
```diff
@@ -11,17 +11,14 @@
...
+const mobile3GThrottling = require('../config/constants').throttling.mobile3G;
 const Util = require('../report/v2/renderer/util.js');
 
-// Maximum TTFI to be considered "fast" for PWA baseline checklist
+// Maximum TTI to be considered "fast" for PWA baseline checklist
 //   https://developers.google.com/web/progressive-web-apps/checklist
-const MAXIMUM_TTFI = 10 * 1000;
-
-const WHITELISTED_STATUS_CODES = [307];
+const MAXIMUM_TTI = 10 * 1000;
```
Comment (paulirish) [inline lighthouse-core/audits/load-fast-enough-for-pwa.js]: Just to call out explicitly: 
We're changing the PWA speed threshold from `TTFI <= 10s` to be a slightly more aggressive `TTI < 10s`.

File: lighthouse-core/audits/load-fast-enough-for-pwa.js [hunk 2]
```diff
@@ -32,116 +29,43 @@ class LoadFastEnough4Pwa extends Audit {
...
+    const settingOverrides = {throttlingMethod: 'simulate', throttling: mobile3GThrottling};
+    const settings =
+      context.settings.throttlingMethod !== 'provided' &&
+      isDeepEqual(context.settings.throttling, mobile3GThrottling)
+        ? context.settings
+        : Object.assign({}, context.settings, settingOverrides);
+    const metricComputationData = {trace, devtoolsLog, settings};
+    const tti = await artifacts.requestConsistentlyInteractive(metricComputationData);
+
+    const score = Number(tti.timing <= MAXIMUM_TTI);
```
Comment (paulirish) [inline lighthouse-core/audits/load-fast-enough-for-pwa.js]: https://developers.google.com/web/progressive-web-apps/checklist does say 

> interactive <10s for first visit on a simulated 3G network.

let's change this to `<` just to simplify. nice to stay single digits. :)

File: lighthouse-core/audits/load-fast-enough-for-pwa.js [hunk 3]
```diff
@@ -32,116 +29,43 @@ class LoadFastEnough4Pwa extends Audit {
...
-        const isFast = timeToFirstInteractive < MAXIMUM_TTFI;
-
-        const extendedInfo = {
-          value: {areLatenciesAll3G, firstRequestLatencies, isFast, timeToFirstInteractive},
-        };
+  static async audit(artifacts, context) {
+    const trace = artifacts.traces[Audit.DEFAULT_PASS];
+    const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    const settingOverrides = {throttlingMethod: 'simulate', throttling: mobile3GThrottling};
+    const settings =
```
Comment (paulirish) [inline lighthouse-core/audits/load-fast-enough-for-pwa.js]: can you drop a comment for this line?

File: lighthouse-core/audits/load-fast-enough-for-pwa.js (hunk 2 above)
Comment (patrickhulce) [reply lighthouse-core/audits/load-fast-enough-for-pwa.js]: sg :)

File: lighthouse-core/audits/load-fast-enough-for-pwa.js (hunk 3 above)
Comment (patrickhulce) [reply lighthouse-core/audits/load-fast-enough-for-pwa.js]: yeah good call done

**Pull request merged**